jobs/build/promote/Jenkinsfile

```diff
@@ -307,7 +307,10 @@ node {
 
     if (major > 3 && ga_release) {
         def next_minor = "${major}.${minor + 1}"
-        if (!(commonlib.ocpReleaseState[next_minor] && commonlib.ocpReleaseState[next_minor]["release"])) {
+        def out = buildlib.doozer("--group=openshift-${next_minor} config:read-group --yaml release_state",
```

Comment:
promote is deprecated. better not touch it

Reply:
Removed with https://github.com/openshift-eng/aos-cd-jobs/pull/3722